test/egaugex_test.exs

```diff
@@ -17,4 +17,28 @@ defmodule EgaugexTest do
     refute Egaugex.Auth.random_string(16) == Egaugex.Auth.random_string(16)
     assert String.length(Egaugex.Auth.random_string(16)) === 16
   end
+
+  test "parse_egauge_data parses egauge data correctly" do
+    data = """
+    <?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
suggest we download files into a "static" folder so we can easily parse "full real files".